pkg/datarepair/datarepair_test.go

```diff
@@ -28,10 +27,10 @@ func TestDataRepair(t *testing.T) {
 		// first, upload some remote data
 		ul := planet.Uplinks[0]
 		satellite := planet.Satellites[0]
-		// stop discovery service so that we do not get a race condition when we delete nodes from overlay cache
-		satellite.Discovery.Service.Discovery.Stop()
-		satellite.Discovery.Service.Refresh.Stop()
-		satellite.Discovery.Service.Graveyard.Stop()
+		// pause discovery service so that we do not get a race condition when we delete nodes from overlay cache
+		satellite.Discovery.Service.Discovery.Pause()
+		satellite.Discovery.Service.Refresh.Pause()
+		satellite.Discovery.Service.Graveyard.Pause()
```

Comment:
It shouldn't make a difference whether you stop or pause.

Reply:
I agree.  But I saw some issues when Stop was used.  Maybe I misread the issue.  I'll change back to Stop and see if I can reproduce again.